editor/src/clj/editor/editor_extensions.clj

```diff
@@ -466,20 +466,59 @@
         root-path (-> project
                       (project/workspace evaluation-context)
                       (workspace/project-path evaluation-context)
-                      .toPath
-                      .normalize)
-        dir-path (-> (str root-path proj-path) io/file .toPath .normalize)]
+                      (fs/as-path)
+                      (fs/to-real-path))
+        dir-path (-> (str root-path proj-path)
+                     (fs/as-path)
+                     (.normalize))]
     (if (.startsWith dir-path root-path)
       (try
         (fs/create-path-directories! dir-path)
         (vreset! request-sync true)
         nil
         (catch FileAlreadyExistsException e
-          (throw (LuaError. (str "File already exists: " (.getMessage e)))))
+          (throw (LuaError. (str "Directory already exists: " (.getMessage e)))))
```

Comment:
I'm fairly certain the error message should say "file already exists" and not "directory already exists". Directory already existing is not an error and should not throw exception. What should throw exception is if you wanted a directory at a certain path, and it turned out at that path there is a file instead, that's what the error it's about.

Reply:
Oh really? I was thinking it's just a typo. So how to fix it back, a new [PR](https://github.com/defold/defold/pull/8481)?